### ISSUE TITLE
Possibility to switch between at home and away mode

### ISSUE THREAD

**Issue by CM-UH** [birkir/homebridge-plugin-eufy-security#24]
I'm using all types of EUFY cam's. They are working fine for me.
Also the plugin works and I have no problem with it (Push Client is off)
But I have a wish, will it be possible to add an option for switching the cams between home and away mode!
The geofencing from EUFY doesn't work and uses a lot of battery.

**Comment by DanielsCode** [comment]
Would be a great feature

**Comment by lpaso** [comment]
Yes, it’s a must have indeed, to be able to manage the security mode via Apple HomeKit instead of Eufy App
